src/Lykke.Service.ExchangeDataStore/Modules/ServiceModule.cs

```diff
@@ -55,13 +58,19 @@ private void BindAzureRepositories(ContainerBuilder container)
                 _settings.ConnectionString(i => i.AzureStorage.EntitiesConnString), _settings.CurrentValue.AzureStorage.EntitiesTableName, _log);
             container.RegisterInstance(orderBookSnapshotStorage).As<INoSQLTableStorage<OrderBookSnapshotEntity>>().SingleInstance();
 
+            var exchangeInstrumentsRepo = AzureTableStorage<ExchangeInstrumentEntity>.Create(_settings.ConnectionString(i => i.AzureStorage.EntitiesConnString), "ExchangeInstruments", _log);
+            container.RegisterInstance(exchangeInstrumentsRepo).As<INoSQLTableStorage<ExchangeInstrumentEntity>>().SingleInstance();
+
             container.RegisterType<OrderBookRepository>().As<IOrderBookRepository>();
```

Comment:
Probably the repositories should be Singleton?

Reply:
Done.